src/components/Attachment/__tests__/File.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
+import { render } from '@testing-library/react';
 import '@testing-library/jest-dom';
-import renderer from 'react-test-renderer';
```

Comment:
`react-test-renderer` has been deprecated. Therefore many snapshots had to be changed.